_data/sites.yml

```
@@ -850,8 +850,8 @@
 
 - domain: enesergun.net
   url: https://enesergun.net
-  size: 205
-  last_checked: 2022-09-19
+  size: 32.2
```

```suggestion
  size: 83.9
```

this should be the uncompressed size.